### PR TITLE
New Dionae Loadout Items - Wave1

### PR DESCRIPTION
**Pull request by Yonnimer**
Adds several new items to the Dionae loadout:
- Three sets of clothes and two hoods for voidtamers
- A jacket, cloak, two hats, and an undersuit for the ekane
- A narrows hardhat and jacket, as well as a slightly reworked jumpsuit (made chonk)
- New Eternal Robes, the old ones are remaining for now but may be reworked in the second wave of items
- Also removed some redundant rokz and serz items and the ships themselves, these may be reworkedin the future, but for now aren't in current lore.

More loadout items will ideally be coming soon for each factions in the coming weeks/months.

### CODE REVIEW

**Comment by Yonnimer** [comment]
!review

File: code/modules/client/preference_setup/loadout/items/xeno/diona.dm [hunk 1]
```diff
@@ -73,6 +152,56 @@ ABSTRACT_TYPE(/datum/gear/suit/diona)
...
+	path = /obj/item/clothing/head/diona/voidtamer/open
+	cost = 1
+	whitelisted = list(SPECIES_DIONA, SPECIES_DIONA_COEUS)
+	sort_category = "Xenowear - Diona"
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/culture/diona_voidtamer)
+
+/datum/gear/suit/voidtamer
+	display_name = "voidtamer jacket"
+	description = "A jacket made of woven carp hide and adorned with golden, typically worn by various voidtamer-alligned factions.."
```
Comment (hazelrat) [inline code/modules/client/preference_setup/loadout/items/xeno/diona.dm:177]:
```suggestion
	description = "A jacket made of woven carp hide and adorned with gold, typically worn by various voidtamer-aligned factions."
```

File: code/modules/client/preference_setup/loadout/items/xeno/diona.dm [hunk 2]
```diff
@@ -73,6 +152,56 @@ ABSTRACT_TYPE(/datum/gear/suit/diona)
...
+	path = /obj/item/clothing/accessory/poncho/voidtamer
+	cost = 1
+	whitelisted = list(SPECIES_DIONA, SPECIES_DIONA_COEUS)
+	sort_category = "Xenowear - Diona"
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/culture/diona_voidtamer)
+
+/datum/gear/suit/voidtamer/apron
+	display_name = "voidtamer apron"
+	description = "A apron made of woven carp hide and adorned with golden, typically worn by various voidtamer-alligned factions.."
```
Comment (hazelrat) [inline code/modules/client/preference_setup/loadout/items/xeno/diona.dm:187]:
```suggestion
	description = "A apron made of woven carp hide and adorned with gold, typically worn by various voidtamer-aligned factions."
```

File: code/modules/client/preference_setup/loadout/items/xeno/diona.dm [hunk 3]
```diff
@@ -73,6 +152,56 @@ ABSTRACT_TYPE(/datum/gear/suit/diona)
...
+	path = /obj/item/clothing/accessory/poncho/voidtamer/apron
+	cost = 1
+	whitelisted = list(SPECIES_DIONA, SPECIES_DIONA_COEUS)
+	sort_category = "Xenowear - Diona"
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION
+	culture_restriction = list(/singleton/origin_item/culture/diona_voidtamer)
+
+/datum/gear/suit/voidtamer/vest
+	display_name = "voidtamer vest"
+	description = "A vest made of woven carp hide and adorned with golden, typically worn by various voidtamer-alligned factions.."
```
Comment (hazelrat) [inline code/modules/client/preference_setup/loadout/items/xeno/diona.dm:197]:
```suggestion
	description = "A vest made of woven carp hide and adorned with gold, typically worn by various voidtamer-aligned factions."
```

File: code/modules/clothing/head/xenos/diona.dm [hunk 4]
```diff
@@ -1,8 +1,38 @@
...
-	icon_state = "rokz_helm"
-	contained_sprite = FALSE
-	species_restricted = list(SPECIES_DIONA, SPECIES_DIONA_COEUS)
+	item_state = "ekanehat"
+	icon_state = "ekanehat"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/diona/ekane/decorated
+	name = "decorated feather cap"
+	desc = "an exuberant hat adorned with colorful feathers across its surface. Despite the design, wearers will feel the heat of the sun greater than without."
```
Comment (hazelrat) [inline code/modules/clothing/head/xenos/diona.dm:11]:
```suggestion
	desc = "An exuberant hat adorned with colorful feathers across its surface. Despite the design, wearers will feel the heat of the sun greater than without."
```

File: code/modules/clothing/head/xenos/diona.dm [hunk 5]
```diff
@@ -1,8 +1,38 @@
-/obj/item/clothing/head/diona/rokz
-	name = "bone helmet"
-	desc = "A large helmet made out of the skull of some kind of large creature."
+/obj/item/clothing/head/diona/ekane
+	name = "feather cap"
+	desc = "a simple hat adorned with colorful feathers across its surface. Despite the design, wearers will feel the heat of the sun greater than without."
```
Comment (hazelrat) [inline code/modules/clothing/head/xenos/diona.dm:3]:
```suggestion
	desc = "A simple hat adorned with colorful feathers across its surface. Despite the design, wearers will feel the heat of the sun greater than without."
```

File: code/modules/clothing/head/xenos/diona.dm [hunk 6]
```diff
@@ -1,8 +1,38 @@
...
+	item_state = "ekane_hatd"
+	icon_state = "ekane_hatd"
+	icon_override = null
+	contained_sprite = TRUE
+	build_from_parts = TRUE
+	has_accents = TRUE
+
+/obj/item/clothing/head/diona/voidtamer
+	name = "carp hood"
+	desc = "a hood made of aged and tanned carp hide, decorated with gold, pointed structures across all sides."
```
Comment (hazelrat) [inline code/modules/clothing/head/xenos/diona.dm:21]:
```suggestion
	desc = "A hood made of aged and tanned carp hide, decorated with gold, featuring pointed structures across all sides."
```

File: code/modules/clothing/head/xenos/diona.dm [hunk 7]
```diff
@@ -1,8 +1,38 @@
...
+	icon = 'icons/obj/diona_items.dmi'
+	contained_sprite = TRUE
+	item_state = "void_hood"
+	icon_state = "void_hood"
+
+/obj/item/clothing/head/diona/voidtamer/open
+	name = "open carp hood"
+	desc = "a hood made of aged and tanned carp hide, decorated with gold, pointed structures across all sides. This one has an open visor, showing the wearer's face."
```
Comment (hazelrat) [inline code/modules/clothing/head/xenos/diona.dm:29]:
```suggestion
	desc = "A hood made of aged and tanned carp hide, decorated with gold, featuring pointed structures across all sides. This one has an open visor, showing the wearer's face."
```

File: code/modules/clothing/head/xenos/diona.dm [hunk 8]
```diff
@@ -1,8 +1,38 @@
...
+	item_state = "void_hood_open"
+	icon_state = "void_hood_open"
+
+/obj/item/clothing/head/hardhat/narrows
+	name = "narrows hard hat"
+	desc = "An old hardhat painted in Hephestus colors, fabric hanging off the sides to protect the wearer's ears."
```
Comment (hazelrat) [inline code/modules/clothing/head/xenos/diona.dm:35]:
```suggestion
	desc = "An old hard hat painted in Hephaestus colors, fabric hanging off the sides to protect the wearer's ears."
```

File: code/modules/clothing/under/xenos/diona.dm [hunk 9]
```diff
@@ -14,3 +14,13 @@
 	desc = "A traditional tunic worn on Hieroaetheria, made up of several colorful fabrics and adorned with a seemingly still-living, bioluminescent Starvine."
 	icon_state = "hieroaetherian_tunic"
 	item_state = "hieroaetherian_tunic"
+
+/obj/item/clothing/under/diona/ekane
+	name = "woven dress"
+	desc = "A blue, seeming high-quality dress with a colorful strap of fabric running along it, typically seen by Dionae hailing from the Hieroaetherian Ekane."
```
Comment (hazelrat) [inline code/modules/clothing/under/xenos/diona.dm:20]:
```suggestion
	desc = "A blue, seemingly high-quality dress with a colorful strap of fabric running along it, typically seen by Dionae hailing from the Hieroaetherian Ekane."
```

**Review by hazelrat — CHANGES_REQUESTED** [review]
Some imperfect grammar, LGTM otherwise.

**Review by hazelrat — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**